elements/pl-file-editor/pl-file-editor.mustache

```diff
@@ -21,8 +21,83 @@ $(function() {
 <div id="file-editor-{{uuid}}" class="mathjax_ignore">
     <input type="hidden" name="{{name}}">
     <div class="card">
-        <div class="card-header">{{file_name}}</div>
+        <div class="card-header d-flex justify-content-between align-items-center">
+             {{file_name}}
+             <button type="button" class="btn btn-sm btn-secondary settings-button" tabindex="-1">
```

Comment:
Is there a reason you set `tabindex` here? A `<button>` should already be tabbable.

Reply:
Yeah this should be removed, I was thinking that some weird behaviors might show up when modal is present, now it seems not necessary.